native/src/components/Layout.tsx

```diff
@@ -6,10 +6,10 @@ const Wrapper = styled.View`
   background-color: ${props => props.theme.colors.backgroundColor};
 `
 
-type AppPropsType = {
+type AppProps = {
```

Comment:
should be named LayoutProps

Reply:
I don't get it... in my local code base it is renamed in that way

Reply:
Then you only need to push it.